src/app/state/rest/index.ts

```diff
@@ -85,11 +85,11 @@ export const useBAG = (bagId: string, options?: Options) => {
   })
 }
 
-export const usePanorama = (lat?: number, lon?: number, options?: Options) => {
+export const usePanorama = (lat?: number, lon?: number, width?: number, radius?: number, options?: Options) => {
   const handleError = useErrorHandler()
   return useApiRequest<{ url: string }>({
     ...options,
-    url: `https://api.data.amsterdam.nl/panorama/thumbnail/?lat=${ lat }&lon=${ lon }&width=438&radius=180`,
+    url: `https://api.data.amsterdam.nl/panorama/thumbnail/?lat=${ lat }&lon=${ lon }&width=${ width }&radius=${ radius }`,
```

Comment:
Maybe a `buildQueryString(params: Record<string, any>): string` would be nice

Reply:
Will make a seperate PR for it. Thanks